src/main/java/taxi/dao/impl/DriverDaoImpl.java

```diff
@@ -102,10 +107,25 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long id = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
-        Driver driver = new Driver();
-        driver.setId(id);
-        driver.setName(name);
-        driver.setLicenseNumber(licenseNumber);
-        return driver;
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
+        return new Driver(id, login,password, name, licenseNumber);
+    }
+
+    @Override
+    public Optional<Driver> findByDriverName(String name) {
```

Comment:
```suggestion
    public Optional<Driver> findByLogin(String login) {
```
Pls rename the method name to findByDriverLogin() or findByLogin() cos the name and login it's different things and this method corresponds to "find a driver by login"